### ISSUE TITLE
Add translatable labels to the audio player

### ISSUE THREAD

**Issue by johnhooks** [wp-blocks/vinyl#13]
## Problem

The `aria-label` and `aria-valuetext` of the `media-chrome` web components are not accessible from outside the web components, making them unable to use `gettext` functions.

## Solution

Use custom versions of the `media-chrome` web components, to provide access to the labels.
